stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>In other to keep the two narratives, I propose to rewrite this parragraph a bit. One saying what the figure shows ("history of the total number of people vaccinated with first and second doses"), and another that says that the plot_cumulative_doses function to builds the figure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>